File: csharp/src/main/CsharpDotNet2/spoonacular/Model/InlineResponse20016.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace spoonacular.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class InlineResponse20016 {
    /// <summary>
    /// Gets or Sets ParsedInstructions
    /// </summary>
    [DataMember(Name="parsedInstructions", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "parsedInstructions")]
    public List<Object> ParsedInstructions { get; set; }

    /// <summary>
    /// Gets or Sets Ingredients
    /// </summary>
    [DataMember(Name="ingredients", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "ingredients")]
    public List<Object> Ingredients { get; set; }

    /// <summary>
    /// Gets or Sets Equipment
    /// </summary>
    [DataMember(Name="equipment", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "equipment")]
    public List<Object> Equipment { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class InlineResponse20016 {\n");
      sb.Append("  ParsedInstructions: ").Append(ParsedInstructions).Append("\n");
      sb.Append("  Ingredients: ").Append(Ingredients).Append("\n");
      sb.Append("  Equipment: ").Append(Equipment).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart/com/spoonacular/client/model/inline_response20016.dart
part of openapi.api;

class InlineResponse20016 {
  
  List<Object> parsedInstructions = [];
  
  List<Object> ingredients = [];
  
  List<Object> equipment = [];
  InlineResponse20016();

  @override
  String toString() {
    return 'InlineResponse20016[parsedInstructions=$parsedInstructions, ingredients=$ingredients, equipment=$equipment, ]';
  }

  InlineResponse20016.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    if (json['parsedInstructions'] == null) {
      parsedInstructions = null;
    } else {
      parsedInstructions = Object.listFromJson(json['parsedInstructions']);
    }
    if (json['ingredients'] == null) {
      ingredients = null;
    } else {
      ingredients = Object.listFromJson(json['ingredients']);
    }
    if (json['equipment'] == null) {
      equipment = null;
    } else {
      equipment = Object.listFromJson(json['equipment']);
    }
  }

  Map<String, dynamic> toJson() {
    Map <String, dynamic> json = {};
    if (parsedInstructions != null)
      json['parsedInstructions'] = parsedInstructions;
    if (ingredients != null)
      json['ingredients'] = ingredients;
    if (equipment != null)
      json['equipment'] = equipment;
    return json;
  }

  static List<InlineResponse20016> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineResponse20016>() : json.map((value) => new InlineResponse20016.fromJson(value)).toList();
  }

  static Map<String, InlineResponse20016> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, InlineResponse20016>();
    if (json != null && json.isNotEmpty) {
      json.forEach((String key, dynamic value) => map[key] = new InlineResponse20016.fromJson(value));
    }
    return map;
  }
}

